### PR TITLE
feat: add simulate button

### PR DESCRIPTION
**Pull request by MeLlamoPablo**
Blocked by #1286 

https://www.notion.so/prepo/Market-Details-simulate-button-6d9e731f7a764725a632a26aaa3146f0

https://user-images.githubusercontent.com/11708035/236257254-6aa7445a-3d9c-4586-b552-3ee6f7cdaa60.mp4

### CODE REVIEW

File: apps/frontend/core-dapp/src/features/trade/MarketDetails.tsx [hunk 1]
```diff
@@ -25,8 +48,27 @@ const MarketDetails: React.FC = () => {
       minDecimals: 0,
     })
 
+  console.log(selectedMarket)
+
```
Comment (chrisling-dev) [inline apps/frontend/core-dapp/src/features/trade/MarketDetails.tsx:52]: remove this?

File: apps/frontend/core-dapp/src/features/simulator/TradeSimulator.tsx [hunk 2]
```diff
@@ -2,14 +2,15 @@ import { useEffect, useMemo } from 'react'
...
   Flex,
   Icon,
   mapDistanceToPrice,
   mapPriceToDistance,
   media,
   spacingIncrement,
-  Tooltip,
-} from 'prepo-ui'
+  Tooltip
+} from "prepo-ui"
```
Comment (chrisling-dev) [inline apps/frontend/core-dapp/src/features/simulator/TradeSimulator.tsx:13]: Is this changed by eslint? Our app uses single quote (enforced by eslint and prettier), it's weird that all other packages is still using single quote except for this one?

```suggestion
} from 'prepo-ui'
```

File: apps/frontend/core-dapp/src/features/trade/MarketDetails.tsx [hunk 3]
```diff
@@ -25,8 +48,27 @@ const MarketDetails: React.FC = () => {
...
   return (
-    <OperationSummary key="summary" header={<span>Market details</span>}>
+    <OperationSummary
+      key="summary"
+      header={
+        <Header>
+          <span>Market details</span>
+          <SimulateButton
+            // Hidden rather than unmounted so there's no height jump
+            $visible={!simulatorOpen && !selectedMarket.resolved}
```
Comment (chrisling-dev) [inline apps/frontend/core-dapp/src/features/trade/MarketDetails.tsx:61]: Would recommend making a computed `showSimulator` state in TradeStore so the same logic can be reused across MarketDetails and TradeSimulator

File: apps/frontend/core-dapp/src/features/trade/MarketDetails.tsx [hunk 4]
```diff
@@ -25,8 +48,27 @@ const MarketDetails: React.FC = () => {
...
+
   return (
-    <OperationSummary key="summary" header={<span>Market details</span>}>
+    <OperationSummary
+      key="summary"
+      header={
+        <Header>
+          <span>Market details</span>
+          <SimulateButton
+            // Hidden rather than unmounted so there's no height jump
```
Comment (chrisling-dev) [inline apps/frontend/core-dapp/src/features/trade/MarketDetails.tsx:60]: I think can remove this comment

**Review by chrisling-dev — CHANGES_REQUESTED** [review]

File: apps/frontend/core-dapp/src/features/simulator/TradeSimulator.tsx (hunk 2 above)
Comment (MeLlamoPablo) [reply apps/frontend/core-dapp/src/features/simulator/TradeSimulator.tsx]: So the problem I had is that my git hooks weren't running for some reason. A simple `husky install` fixed it, so I shouldn't have these kind of problems moving forward.

**Review by chrisling-dev — APPROVED** [review]
LGTM!

**Pull request merged**